riscv/supportcoderiscv.py

```diff
@@ -361,13 +409,13 @@ def parse_flag(argv, flagname):
 helptext = """
 Usage: %s [options] <elf_file>
```

Comment:
is the dtb an elf file? If not, this might be confusing
```suggestion
Usage: %s [options] <binary_file>

Run the pydrofoil emulator on a file, which can be a bootable
device-tree-blob or a compiled elf executable
```

Reply:
no, the 'bbl' format that the linux kernel is in is an elf file, weirdly enough. it has a 'bbl' ending, which stands for "berkeley boot loader" or so?

But it's a good idea to also add a sentence to the `--help` about what will happen, will do that in a separate commit.